apps/heureka/src/components/shared/Helper.jsx

```diff
@@ -45,3 +45,36 @@ export const severityString = (severity) => {
 
   return severity?.value + " (" + severity?.score + ")"
 }
+
+// Helper function to extract the last four parts of a URL - used for displaying ccrn
+export const extractLastFourParts = (url) => {
```

Comment:
Where else you plan to use this helper?

Reply:
I removed it from here, put it inside the component which uses it.